systems/controllers/osc/operational_space_control.cc

```diff
@@ -381,8 +393,18 @@ VectorXd OperationalSpaceControl::SolveQp(
     const VectorXd& x_w_spr, const VectorXd& x_wo_spr,
     const drake::systems::Context<double>& context, double t, int fsm_state,
     double time_since_last_state_switch) const {
-  vector<bool> active_contact_flags = CalcActiveContactIndices(fsm_state);
+  // Get active contact indices
+  vector<int> active_contact_list = {};
```

Comment:
Because we only use this to check if a contact index exists in the active set, this really should be a `std::set<int>` instead of a `std::vector<int>`

Reply:
Correct, I didn't realize what I needed was `std::set`